browser/src/control/Control.DownloadProgress.js

```diff
@@ -150,7 +153,7 @@ L.Control.DownloadProgress = L.Control.extend({
 	},
 
 	setupKeyboardShortcutForSnackbar: function () {
-		this._setupKeyboardShortcutForElement('snackbar', 'button');
+		this._setupKeyboardShortcutForElement('snackbar-container', 'button');
```

Comment:
OK, but why is this needed? Not clear how we had snackbar here in the past and how snackbar-container is better now.

Reply:
Narrows down the search. Not vital though :)